CDCgov/NEDSS-Modernization
# CNFT2-1153 api swap

## Description

Removes the GET `/pages/{page}/details` API and changes the Page Builder UI to use the GET `pages/{page}` API

## Tickets

* [CNFT2-1153](https://cdc-nbs.atlassian.net/browse/CNFT2-1153)

## Steps to verify

1. Start Page Builder
2. Navigate to a Page
3. Ensure that the Page is displayed


[CNFT2-1153]: https://cdc-nbs.atlassian.net/browse/CNFT2-1153?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
